frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -30,6 +30,10 @@
     text-decoration: none;
 }
 
+.metadata .comment .content {
+    display: inline-block;
+}
+
```

Comment:
Just realized this change is breaking comments into new lines:
https://mozilla-pontoon-staging.herokuapp.com/sl/firefox/all-resources/?string=74127

Why do we need it?

Reply:
Since the pinned comments are coming in as `<p></p>` those comments were breaking into 2 lines without this. What's interesting is that locally `Comment` is not split:

<img width="600" alt="Screen Shot 2020-09-25 at 1 15 10 PM" src="https://user-images.githubusercontent.com/19564687/94312389-a3836880-ff31-11ea-81b3-3f50511f7895.png">

It looks like if I move it to the `<p>` then both comments render correctly. Let me know if that is what I should do or if you have an alternate solution.

Reply:
SGTM